docs/EN_US/ContainerizedHPCC/ContainerizedMods/TerraformDeployChap.xml

```diff
@@ -0,0 +1,568 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE section PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd">
+<chapter id="HPCC_Terraform_deployment">
+  <title>Deploying HPCC Systems<superscript>®</superscript> with
+  Terraform</title>
+
+  <para>Manual deployments can be error-prone and inconsistent. As your
+  deployments become more customized and your need for additional resources
+  grows it can become exponentially more difficult and time consuming.</para>
+
+  <para>There is good news though, there are multiple IaC (infrastructure as
```

Comment:
"There is good news though," is not in our usual voice for books. It is good for a blog, but not so much here.